.gitlab-ci.yml

```diff
@@ -44,27 +44,41 @@ test:
   stage: test
   tags:
     - kube
-  script:
+  allow_failure: false
+  before_script:
     - docker pull $REPO_URL/$BASE_IMAGE_NAME:$CI_COMMIT_SHA
+  script:
     - docker run $REPO_URL/$BASE_IMAGE_NAME:$CI_COMMIT_SHA ./test.sh
 
 test-lint:
   stage: test
   tags:
     - kube
-  allow_failure: true
-  script:
+  allow_failure: false
```

Comment:
allow_failure "false" is the default for gitlab ci, so you can omit this statement